crates/admin/src/cluster_controller/service.rs

```diff
@@ -386,6 +402,26 @@ impl<T: TransportConnect> Service<T> {
                 node.partitions
                     .get(&partition_id)
                     .filter(|status| status.is_effective_leader())
+                    .inspect(|status| {
```

Comment:
Seemed like a good idea at the time but this doesn't add much of value. May remove it before merging.

Reply:
+1 to removing